decentraland/unity-renderer
# Feat: added top chat notification

## What does this PR change?

Fix #2401 
Adds top notification

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/top-notification&ENABLE_NOTIFICATION_CHAT
2. Play with chat and notification
3. Verify that everything works as expected
4. Verify that when chat or channels are opened a top chat appears
5. Verify that it updates when new notifications arrive
6. Verify that by clicking the notification it opens the correct chat

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
